src/config/account1.js

```diff
@@ -2,22 +2,17 @@ module.exports = {
   account: {
     username: 'username', // Steam username
     password: 'password', // Steam password
-    statusInvisible: false, // If set to "true" friends won't see notification or you playing anything
+    statusInvisible: false, // If "true" friends won't see notification or you playing anything
     shared_secret: '' // Auto 2FA login
   },
   // Below settings are optional if they are supplied in the global config
   idlerSettings: {
+    enabled: true, // Turn idler on or off
     parallelGameIdle: 32, // Amount of games playing at the same time (max is 32)
-    staticIdleTime: 0 // Number of min to idle for before switching (0 is random number)
-  },
-  staticIdler: {
-    enabled: false, // Idle static list of games (listed in "listToIdle")
-    listToIdle: [] // List to idle (example: [730, 570, 440])
-  },
-  dynamicIdler: {
-    enabled: false, // Idle random list of games (randomly selected from the game you own)
-    skipBannedGames: false, // If "true" it won't idle games you're banned in
-    skipFreeGames: false, // If "true" it won't idle free to play games
-    blacklistGames: [] // List of games not to idle (example: [730, 570, 440])
+    staticIdleTime: 0, // Number of min to idle for before switching games (0 means randomized number)
+    staticIdleList: [], // Games that will always be idled, example: [730, 570, 440]
```

Comment:
Thoughts on changing these?
```suggestion
    idleTime: 0, // Number of min to idle for before switching games (0 means randomized number)
    alwaysIdleList: [], // Games that will always be idled, example: [730, 570, 440]
```

Reply:
Changed it